### DISCUSSION TITLE
The link looks clickable, but the browser did not open

### DISCUSSION THREAD

**Discussion by RigottiG** [wezterm/wezterm#3947] [Q&A] [answered]
I am facing a problem when I try to open a link in wezterm, I click on the link, but it does not open the browser.

`.wezterm.lua`

```lua
local wezterm = require("wezterm")
local config = wezterm.config_builder()

config.default_prog = { "tmux" }
config.color_scheme = "Gruvbox (Gogh)"

config.font = wezterm.font_with_fallback({ "JetBrainsMono Nerd Font", "Broot Icons Visual Studio Code" })
config.font_size = 11

config.enable_tab_bar = false

config.initial_rows = 36
config.initial_cols = 146

config.disable_default_key_bindings = true

local act = wezterm.action
config.keys = {
	{ key = ")", mods = "CTRL", action = act.ResetFontSize },
	{ key = "-", mods = "CTRL", action = act.DecreaseFontSize },
	{ key = "=", mods = "CTRL", action = act.IncreaseFontSize },
	{ key = "N", mods = "CTRL", action = act.SpawnWindow },
	{ key = "P", mods = "CTRL", action = act.ActivateCommandPalette },
	{ key = "V", mods = "CTRL", action = act.PasteFrom("Clipboard") },
	{ key = "Copy", mods = "NONE", action = act.CopyTo("Clipboard") },
	{ key = "Paste", mods = "NONE", action = act.PasteFrom("Clipboard") },
	{ key = "F11", mods = "NONE", action = act.ToggleFullScreen },
}

return config
```

**Comment by nekowinston** [comment]
Can you have a look at the debug overlay (CTRL + SHIFT + L) to see what it logs?

It shows `INFO wezterm_gui::termwindow > clicking https://example.com` for me when the browser opens.

If you're on linux, `xdg-open https://example.com` should work in the terminal.

**Reply by RigottiG** [reply]
![image](https://github.com/wez/wezterm/assets/13246770/3c806092-1336-429b-ae48-2ad11cd8a11f)

the log does not work. I click on example.com and not happen.

I tried to run 'print https://www.google.com' on REPL in Debug mode and it works 👀

**Reply by nekowinston** [reply]
:thinking: I see that you're using tmux, does it work if you click on links outside tmux?

**Reply by RigottiG** [reply]
Unexpectedly yes, it seems that the problem is in tmux then.

**Reply by nekowinston** [reply, marked as answer]
https://wezfurlong.org/wezterm/config/mouse.html#gotcha-on-binding-an-up-event-only

https://wezfurlong.org/wezterm/config/mouse.html#default-mouse-assignments

It seems like "Single Left Up" is bound to `act.CompleteSelectionOrOpenLinkAtMouseCursor("ClipboardAndPrimarySelection")` by default, which won't be returned by tmux.

**Reply by wez** [reply]
If you have enabled mouse mode in tmux, then mouse events are sent to tmux rather than being processed as mouse assignments. See the 3rd and 4th paragraphs of https://wezfurlong.org/wezterm/config/mouse.html for more information about this, and also: https://wezfurlong.org/wezterm/config/lua/config/bypass_mouse_reporting_modifiers.html to control which modifier keys you can use to force your wezterm mouse assignments to be processed.
